### ISSUE TITLE
Please add our paper

### ISSUE THREAD

**Issue by wgcban** [MinZHANG-WHU/Change-Detection-Review#3]
Hi,

Can you please add our recent CD paper with Transformers ("A Transformer-Based Siamese Network for Change Detection") to your collection?

arxiv link: https://arxiv.org/abs/2201.01293
Code: https://github.com/wgcban/ChangeFormer
Abstract: This paper presents a transformer-based Siamese network architecture (abbreviated by ChangeFormer) for Change Detection (CD) from a pair of co-registered remote sensing images. Different from recent CD frameworks, which are based on fully convolutional networks (ConvNets), the proposed method unifies hierarchically structured transformer encoder with Multi-Layer Perception (MLP) decoder in a Siamese network architecture to efficiently render multi-scale long-range details required for accurate CD. Experiments on two CD datasets show that the proposed end-to-end trainable ChangeFormer architecture achieves better CD performance than previous counterparts.

Thank you.

**Comment by wgcban** [comment]
Also this:

Revisiting Consistency Regularization for Semi-supervised Change Detection in Remote Sensing Images

Paper: [https://arxiv.org/abs/2204.08454](https://arxiv.org/abs/2204.08454)
Code: [https://github.com/wgcban/SemiCD](https://github.com/wgcban/SemiCD)

Abstract: Remote-sensing (RS) Change Detection (CD) aims to detect "changes of interest" from co-registered bi-temporal images. The performance of existing deep supervised CD methods is attributed to the large amounts of annotated data used to train the networks. However, annotating large amounts of remote sensing images is labor-intensive and expensive, particularly with bi-temporal images, as it requires pixel-wise comparisons by a human expert. On the other hand, we often have access to unlimited unlabeled multi-temporal RS imagery thanks to ever-increasing earth observation programs. In this paper, we propose a simple yet effective way to leverage the information from unlabeled bi-temporal images to improve the performance of CD approaches. More specifically, we propose a semi-supervised CD model in which we formulate an unsupervised CD loss in addition to the supervised Cross-Entropy (CE) loss by constraining the output change probability map of a given unlabeled bi-temporal image pair to be consistent under the small random perturbations applied on the deep feature difference map that is obtained by subtracting their latent feature representations. Experiments conducted on two publicly available CD datasets show that the proposed semi-supervised CD method can reach closer to the performance of supervised CD even with access to as little as 10% of the annotated training data.
